#879 Add Project Owner to 'Project Member Can't Create Namespace' doc (#894)
rancher/rancher-docs: versioned_docs/version-2.7/reference-guides/rancher-webhook.md

## What Resources Does the Webhook Validate?

> An in-progress list of the resources that the webhook validates can be found in the [webhook's repo](https://github.com/rancher/webhook/blob/release/v0.4/docs.md). These docs are organized by group/version and resource (top-level header is group/version, next level header is resource). Checks specific to one version can be found by viewing the `docs.md` file associated with a particular tag (note that webhook versions prior to `v0.3.6` won't have this file).

Reviewer: ```suggestion
You can find an in-progress list of the resources that the webhook validates in the [webhook's repo](https://github.com/rancher/webhook/blob/release/v0.4/docs.md). These docs are organized by group/version and resource (top-level header is group/version, next level header is resource). Checks specific to one version can be found by viewing the `docs.md` file associated with a particular tag (note that webhook versions prior to `v0.3.6` won't have this file).
```

> You can find an in-progress list of the resources that the webhook validates in the [webhook's repo](https://github.com/rancher/webhook/blob/release/v0.4/docs.md). These docs are organized by group/version and resource (top-level header is group/version, next level header is resource). Checks specific to one version can be found by viewing the `docs.md` file associated with a particular tag (note that webhook versions prior to `v0.3.6` won't have this file).

Reviewer 2: The changes in this file don't seem to be synced to the main page in `docs/reference-guides/rancher-webhook.md`. Would you mind copying the changes over to the main file as well?

## Bypassing the Webhook

> Sometimes, bypassing Rancher's webhook validation may be necessary to perform emergency restore operations or fix other critical issues. The bypass operation is exhaustive, meaning no webhook validations or mutations will apply when using it. It is not possible to bypass some validations or mutations and have others still apply - they are either all bypassed or all active.

Reviewer 2: The changes in this file don't seem to be synced to the main page in `docs/reference-guides/rancher-webhook.md`. Would you mind copying the changes over to the main file as well?

> Sometimes, it may be necessary to bypass Rancher's webhook validation to perform emergency restore operations, or fix other critical issues. The bypass operation is exhaustive, meaning that no webhook validations or mutations will apply when this is used. It is not possible to bypass some mutations or validations and have others still apply - they are either all bypassed, or all active. 

Reviewer: ```suggestion
Sometimes, bypassing Rancher's webhook validation may be necessary to perform emergency restore operations or fix other critical issues. The bypass operation is exhaustive, meaning no webhook validations or mutations will apply when using it. It is not possible to bypass some validations or mutations and have others still apply - they are either all bypassed or all active.
```

``` bash
helm repo add rancher-charts https://charts.rancher.io 
helm upgrade --reuse-values rancher-webhook rancher-charts/rancher-webhook  -n cattle-system --set global.hostNetwork=true
```
**Note:** This temporary workaround may violate an environment's security policy. This workaround also requires that port 9443 is unused on the host network.

> **Note:** Helm, by default, uses a type that some webhook versions validate (secrets) to store information. In these cases, it's recommended to first directly update the deployment with the hostNetwork=true value using kubectl, and then perform the helm commands listed above to avoid drift between the helm configuration and the actual state in the cluster.

Reviewer: ```suggestion
**Note:** Helm, by default, uses a type that some webhook versions validate (secrets) to store information. In these cases, we recommend directly updating the deployment with the `hostNetwork=true` value first using kubectl, then performing the helm commands listed above to avoid drift between the helm configuration and the actual state in the cluster.
```

Reviewer: @martyav Should the **Note** on line 66 be combined with this **Note**?

> **Note:** Helm, by default, uses a type that some webhook versions validate (secrets) to store information. In these cases, it's recommended to first directly update the deployment with the hostNetwork=true value using kubectl, and then perform the helm commands listed above to avoid drift between the helm configuration and the actual state in the cluster.

Author: ```suggestion
**Note:** Helm uses secrets by default. This is a datatype that some webhook versions validate to store information. In these cases, directly update the deployment with the hostNetwork=true value using kubectl, then perform the helm commands listed above to avoid drift between the helm configuration and the actual state in the cluster.
```

To help alleviate these issues, you can run the [adjust-downstream-webhook](https://github.com/rancherlabs/support-tools/tree/master/adjust-downstream-webhook) shell script after roll back. This script selects and installs the proper webhook version (or removes the webhook entirely) for the corresponding Rancher version. 

> ### Project Members or Owners Can't Create Namespaces 

Author: ```suggestion
### Project Users Can't Create Namespaces 
```
